ts/main.ts

```diff
@@ -21,14 +28,96 @@ $urlLinkInput.addEventListener('input', (event: Event) => {
 $entryForm.addEventListener('submit', (event: Event) => {
   event.preventDefault();
   const $formElements = $entryForm.elements as FormElements;
-  const $formObject = {
+  const $formObject: FormObject = {
     title: $formElements.title.value,
     url: $formElements.url.value,
     notes: $formElements.notes.value,
     entryID: data.nextEntryId,
   };
   data.nextEntryId++;
   data.entries.unshift($formObject);
+  const newEntry = renderEntry($formObject);
+  const $ul = document.querySelector('ul');
```

Comment:
I would move this to the global space. `querySelector`s are inefficient, and this would cause us to do the search everytime this function gets called. If we do it globally, we only need to do the search once, and always can reference it.